.github/workflows/ci.yaml

```diff
@@ -144,7 +144,7 @@ jobs:
 
   publish-docs:
     name: Publish documentation website to GitHub pages
-    needs: ["nox", "build"]
+    needs: ["nox", "build", "protolint"]
```

Comment:
This check makes a lot of sense. Would it make sense to add the `protolint` job as a pre-requisite to `build` job? It looks like the build job need not run if we know the protolint check has failed.

Reply:
I think it is good to have the build run even if linting fails, so we can detect issues in both building and linting at the same time, if that is the case. If we add a dependency there then we'll first get a linting error, we'll fix it, push again only to find out the build fails too.